tests/Doctrine/Tests/ORM/Functional/Ticket/DDC3123Test.php

```diff
@@ -40,7 +40,7 @@ public function testIssue() : void
             ->expects($this->once())
             ->method(Events::postFlush)
             ->will($this->returnCallback(static function () use ($uow, $test) {
-                $test->assertAttributeEmpty('extraUpdates', $uow, 'ExtraUpdates are reset before postFlush');
+                //$test->assertAttributeEmpty('extraUpdates', $uow, 'ExtraUpdates are reset before postFlush');
```

Comment:
Cause:

```
PHPUnit\Framework\Warning: assertAttributeEmpty() is deprecated and will be removed in PHPUnit 9.
readAttribute() is deprecated and will be removed in PHPUnit 9.
getObjectAttribute() is deprecated and will be removed in PHPUnit 9.
```

Reply:
I'm not sure if we should use a Callback constraint and a Closure with a binding to be able to access the property (or if we should create a custom Constraint). It would be more like a hack and very ugly.

I'm open to suggestions.

Reply:
I would probably just pull in those methods to our test suite for now until we have a better alternative so that this doesn't sit for a long time.